star-gateway/HIL_SIMULATION.md

```
@@ -0,0 +1,299 @@
+
+## Quick Start
+
+### Terminal 1: Start Virtual RX72N
+
+```bash
+cd star-gateway
+go run ./cmd/virtual_rx72n/
+```
+
+**Expected Output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)